scripts/EN/Harry

```diff
@@ -7310,15 +7310,15 @@ OP_0x29(); // 0x9458: 00000029
 CleanScreen(); // 0x945c: 00000064
 LoadBackground(0xab); // 0x9460: 00001069
 DrawBackground(); // 0x9466: 00000065
-ShowText((3001)"¥"All right...¥" That is as far as my memory goes. Any thing beyond that is vague, and to give an account of it I would have to make things up."); // 0x946a: 0000100c
+ShowText((3001)"¥"All right…¥"¥nW5That is as far as my memory goes.¥nW2Any thing beyond that is vague, W1and to give an account of it I would have to make things up."); // 0x946a: 0000100c
 HandleInput(); // 0x9470: 00000010
-ShowText((3002)"However, I do not wish to include any fabrications in this story, that is my true intention. In this case truth is stranger than fiction. But if I said that it was the truth then I doubt anyone would believe me."); // 0x9474: 0000100c
+ShowText((3002)"However, I do not wish to include any fabrications in this story, W1that is my true intention.¥nW2In this case W1truth is stranger than fiction.¥nW2But if I said that it was the truth then I doubt anyone would believe me."); // 0x9474: 0000100c
 HandleInput(); // 0x947a: 00000010
-ShowText((3003)"I carefully read over the manuscript again. I could never even think about forgetting the detestable events of seven years ago. Even now, my body quivers."); // 0x947e: 0000100c
+ShowText((3003)"I carefully read over the manuscript again.¥nW2I could never even think about forgetting the detestable events of seven years ago.¥nW2Even now, my body quivers."); // 0x947e: 0000100c
 HandleInput(); // 0x9484: 00000010
-ShowText((3004)"And because of this when the request came in, at first I considered refusing because the contents were too similar to the world view. However, the person in charge pushed me into it."); // 0x9488: 0000100c
+ShowText((3004)"This is why I initially considered refusing when the request came in, W1as the details of the assignment were a bit too similar to what I had gone through.¥nW2However, the person in charge pushed me into it."); // 0x9488: 0000100c
```

Comment:
This is a fairly big change, and I cannot find a way to literally translate, so the result is an interpretation.

The bit that changed is what lies between W1 and W2. Here is an explanation of what it means based on my discussions with native Japanese speakers.

Literally the difficult part is something like "The details of the request were very similar to my own outlook on the world." Which doesn't make a lot of sense in straight English.

To break it down, Harry receives a request to write a book from his publishing company.

Now we can infer from 3005 that people don't know about what happened to him 7 years ago, but we can infer that the request just happened to be something like "write a novel about losing your child" or "write a novel about getting lost in the otherworld".

While we don't know the exact details of the request, Harry wanted to refuse because he knew they would make him remember what he went through in Silent Hill.

Also, I kind of hate using "person in charge." If you have a good word for the person who would give out assignments at a publishing company that would work well. I have debated using "editor", but I don't think that is quite right.